inc/BrainPadDisplay.h

```diff
@@ -0,0 +1,25 @@
+#ifndef BPDISPLAY_H
+#define BPDISPLAY_H
+
+#include "Image.h"
+#include "MbedI2C.h"
+#include "stdint.h"
+
+namespace codal {
+   class BrainPadDisplay {  
+  
+		uint8_t data[2];
+		int DeviceAddress = 0x78;
```

Comment:
This is the I2C bus address, so it should be a private static const. Separate from data.